pkg/processor/native/processor_test.go

```diff
@@ -62,6 +63,68 @@ func (s *BildProcessorSuite) TestBildProcessor_Grayscale() {
 	assert.EqualValues(s.T(), actual, expected)
 }
 
+func (s *BildProcessorSuite) TestBildProcessor_Flip() {
+	var actual, expected []byte
+	var err error
+	out := s.processor.Flip(s.srcImage, "v")
+	actual, err = s.processor.Encode(out, "jpeg")
+	assert.NotNil(s.T(), actual)
+	assert.Nil(s.T(), err)
+	expected, err = ioutil.ReadFile("_testdata/test_flipedV.jpg")
+	assert.NotNil(s.T(), expected)
+	assert.Nil(s.T(), err)
+	assert.EqualValues(s.T(), actual, expected)
+
+	out = s.processor.Flip(s.srcImage, "h")
+	actual, err = s.processor.Encode(out, "jpeg")
+	assert.NotNil(s.T(), actual)
+	assert.Nil(s.T(), err)
+	expected, err = ioutil.ReadFile("_testdata/test_flipedH.jpg")
+	assert.NotNil(s.T(), expected)
+	assert.Nil(s.T(), err)
+	assert.EqualValues(s.T(), actual, expected)
+
+	out = s.processor.Flip(s.srcImage, "vh")
+	actual, err = s.processor.Encode(out, "jpeg")
+	assert.NotNil(s.T(), actual)
+	assert.Nil(s.T(), err)
+	expected, err = ioutil.ReadFile("_testdata/test_flipedVH.jpg")
+	assert.NotNil(s.T(), expected)
+	assert.Nil(s.T(), err)
+	assert.EqualValues(s.T(), actual, expected)
+}
+
+func (s *BildProcessorSuite) TestBildProcessor_Rotate() {
```

Comment:
the same as above, can you please refactor this so that's it's easier to read and less duplication?